sdk/communication/azure-communication-chat/azure/communication/chat/_shared/response.py

```diff
@@ -0,0 +1,8 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+def return_response(response, deserialized, _):  # pylint: disable=unused-argument
```

Comment:
It also seems it is not being used anywhere?

Reply:
Yes, this one is not used since we move the invalid participants into response body. @sarkar-rajarshi Would you mind helping me clean up it?

Reply:
Removed